File: semestr-8/Jezyk C#/alexander/Csharp/Archiwum/2021s/Id06IO1/Command/Command/Sportsmen.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Command
{
    public class Sportsmen
    {
        private string name;
        public Sportsmen(string name) { this.name=name; }
        public string Run() { return "Runing"; }
        public string Swim() { return "Swimming"; }
        public string Jump() { return "Jumping"; }
    }
}

File: semestr-8/Programowanie obiektowe w Java/alexander/POJ/Archiwum/2020w/ID07IO1AB/Command/src/Sportsmen.java
public class Sportsmen
{
   private final String Name;
   public Sportsmen(String Name) { this.Name=Name; }
   String run() { return "Running"; }
   String jump() { return "Jumping"; }
   String swim() { return "Swimming"; }
   @Override public String toString() { return Name; }
}
